metric.go

```diff
@@ -290,3 +247,30 @@ func ValidateTags(tags []string) bool {
 
 	return true
 }
+
+func writeSortedTagString(w io.Writer, name string, tags []string) error {
```

Comment:
Will we be performing tag validation here? Or leave that up to tsdb-gw ?

Reply:
There is a separate function to validate the tags, this is only supposed to write the name with tags into a writer